server/src/main/java/org/opentosca/toscana/plugins/cloudformation/CloudFormationModule.java

```diff
@@ -264,9 +285,65 @@ private String getRandomStackName() {
         return "toscana-stack-" + UUID.randomUUID();
     }
 
+    /**
+     Returns the <tt>Authentication<tt> to access S3 for this module.
```

Comment:
This class is too big. Please extract some aspects into their own class. E.g., this whole authentication aspect can get extracted.

Reply:
I extracted some methods that are responsible for similar things into their own utility classes.